frontend/benefit/applicant/src/pages/login.tsx

```diff
@@ -1,3 +1,4 @@
+import useLogin from 'benefit/handler/hooks/useLogin';
```

Comment:
i would keep it separated just because its very different login. Now the code is shared but still AD and suomi.fi are different. and we might need more user claims in future. But i am not sure

Reply:
typo, you are right